tests/phpunit/tests/fonts/font-face/wp-font-face-tests-dataset.php

```diff
@@ -403,4 +403,94 @@ public static function get_custom_font_families( $key = '' ) {
 
 		return $data;
 	}
+
+	public static function get_custom_style_variations( $key = '' ) {
+		static $data = null;
+
+		$uri                    = get_stylesheet_directory_uri() . '/assets/fonts/';
+		$expected_font_families = array(
+			array(
+				array(
+					'src'          => array(
+						'/var/www/tests/phpunit/data/themedir1/fonts-block-theme/assets/fonts/dm-sans/DMSans-Regular.woff2',
+					),
```

Comment:
```suggestion
					'src'          => array( $uri . 'dm-sans/DMSans-Regular.woff2' ),
```
To help these tests pass in other environments, I'd suggest prefixing with the local `$uri`, similar to [another test above](https://github.com/matiasbenedetto/wordpress-develop/blob/load-fonts-from-style-variations/tests/phpunit/tests/fonts/font-face/wp-font-face-tests-dataset.php#L254).

I won't mark the remaining items below...this applies to each expected result.

Reply:
Resolved 6a70c04c02a9d7cc7cd3398ed53e0dd73de4848b